addon/services/flexberry-logging.js

```diff
@@ -14,22 +12,66 @@ const { getOwner } = Ember;
  */
 export default Ember.Service.extend({
   /**
-    Store for transmit messages to server.
+       Store for transmit messages to the server
+
+       @property {Object} _flexberryStore
+       @private
+       @type DS.Store
+   */
+  _flexberryStore: Ember.inject.service('store'),
+  /**
+       Logger switcher on/off true if remote logging service works correctly
+
+       @property {Boolean} _serverLogEnabled
+       @private
+       @type Boolean
+       @default true
+   */
+  _serverLogEnabled: true,
+  /**
+       Current log level. Set this property to log appropriate interval of log types
+
+       @property {Integer} flexberryLogLevel
+       @type Integer
+       @default 0
+   */
+  flexberryLogLevel: 0,
+
+  /**
+     Log level enumarator
 
-    @property flexberryStore
-    @type DS.Store
-    @default null
+     @property logLevelEnums
+     @type Object
+     @@default { ERROR: 1,WARN: 2, LOG: 3, INFO: 4, DEBUG: 5, DEPRECATION: 6 }
    */
-  flexberryStore: null,
+  logLevelEnums: {
+
+    // Log only errors.
+    ERROR: 1,
+
+    // Log warnings and errors.
+    WARN: 2,
+
+    // Log logs, warnings and errors.
+    LOG: 3,
+
+    // Log infos, logs, warnings and errors.
+    INFO: 4,
 
+    // Log debugs, infos, logs, warnings and errors.
+    DEBUG: 5,
+
+    // Log deprecations, debugs, infos, logs, warnings and errors.
+    DEPRECATION: 6
+  },
   /**
-    Logger switcher on/off true if remote logging service works correctly.
+     Inverted to logLevelEnums array
 
-    @property serverLogEnabled
-    @type Boolean
-    @default true
+     @property enumsLoglevel
+     @type Array
+     @default ['NONE','ERROR','WARN','LOG','INFO','DEBUG','DEPRECATION']
```

Comment:
There is another default value.
What does "Inverted to logLevelEnums array" mean?